demo/glfw_opengl3/Makefile

```diff
@@ -2,7 +2,7 @@
 BIN = demo
 
 # Flags
-CFLAGS += -std=c89 -Wall -Wextra -pedantic
+CFLAGS += -g -std=c89 -Wall -Wextra -pedantic
```

Comment:
I'm okay with this being here :shrug: you can remove if you want but meh

Reply:
I'll leave it.  Honestly I'm always adding it to whatever demo I'm using to test my changes. It's already in 2 of the Makefiles  (probably from me), and it doesn't really make sense not having it for non-release/production development demos anyway.